### PR TITLE
Cleanup music

### PR DESCRIPTION
**Pull request by Discordius**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1206995763896559) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/client/start.tsx [hunk 1]
```diff
@@ -8,6 +8,10 @@ import { createApolloClient } from './apolloClient';
 import { fmCrosspostBaseUrlSetting } from "../lib/instanceSettings";
 import { populateComponentsAppDebug } from '../lib/vulcan-lib';
 import { initServerSentEvents } from "./serverSentEventsClient";
+import ReactJkMusicPlayer from 'react-jinke-music-player'
+import 'react-jinke-music-player/assets/index.css'
+import NoSSR from 'react-no-ssr';
+import Helmet from 'react-helmet';
```
Comment (b0b3rt) [inline packages/lesswrong/client/start.tsx]: (I am not sure if this PR is attempting to fully revert the changes currently only in lw-deploy, but in either case, one of Jim's recent PRs internalized `NoSSR` and `Helmet` to some degree for React 18 typing compatibility, so they should be imported from their new locations)

Comment (Discordius) [reply packages/lesswrong/client/start.tsx]: moot since the imports are gone

File: packages/lesswrong/themes/globalStyles/globalStyles.ts [hunk 2]
```diff
@@ -18,10 +18,10 @@ const clearStyle = (theme: ThemeType): JssStyles => ({
   },
   body: {
     margin: 0,
-    backgroundColor: theme.palette.background.default,
+    background: theme.palette.background.default,
     
     "@media print": {
-      backgroundColor: theme.palette.panelBackground.default,
+      background: theme.palette.panelBackground.default,
```
Comment (b0b3rt) [inline packages/lesswrong/themes/globalStyles/globalStyles.ts:24]: What do these changes do (and are they safe to have un-gated)?

**Review by b0b3rt — APPROVED** [review]
